apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/post-content/index.js

```diff
@@ -19,13 +20,20 @@ registerBlockType( 'a8c/post-content', {
 	icon: 'layout',
 	category: 'layout',
 	supports: {
+		align: [ 'full' ],
```

Comment:
what if we used 

```
		getEditWrapperProps() {
			return { 'data-align': 'full' };
		},
```

like we do for templates? I know it doesn't super matter since we hide the UI with CSS, but we originally removed this so that the user couldn't change it